src/Tickets/REST/TEC/V1/Traits/With_Filtered_Ticket_Params.php

```diff
@@ -161,6 +162,11 @@ public function filter_upsert_params( array $params ): array {
 			throw new InvalidRestArgumentException( __( 'The ticket price must be a number.', 'event-tickets' ) );
 		}
 
+		// Ensure menu_order value is an integer.
```

Comment:
Not needed since this is done for you by the sanitizer of the integer parameter

Reply:
`menu_order` is being handled by the `$ticket_params` and not the `$post_params`.

you need to set it inside the `$new_params` further above and then unset it from the `$post_params`

```php
$new_params['ticket_menu_order'] = $params['menu_order'] ?? null;
unset( $params['menu_order'] );
```

Reply:
> `menu_order` is being handled by the `$ticket_params` and not the `$post_params`.

@dpanta94 This doesn't make sense to me. The `menu_order` parameter should belong to the post object, so it should go with the other `$post_params`.

Why would it be handled differently?